### DISCUSSION TITLE
How to use Slack Plugin in Production Environment

### DISCUSSION THREAD

**Discussion by Saauan** [jongpie/NebulaLogger#306] [Q&A] [answered]
Hello, I discovered this wonderful library a few weeks ago, and I began using it in my app. But when I tried to deploy to my production org (DevHub), I got an error telling me the slack plugin was still a beta package, and it refused me from installing it.

Do you know a way I could still install and use the slack plugin on my production environment or is it really unsafe for me to do so ?

Thank you in advance for the help

**Comment by jongpie** [comment, marked as answer]
Hi @Saauan - sorry about that! I have not promoted the package yet, but I'll plan to do that within the next few days. Once the package has been promoted, you will be able to install it into production. I'll let you know as soon as it's done!

**Reply by Saauan** [reply]
Alright, thank you a lot ! :D

**Reply by jongpie** [reply]
@Saauan I've promoted the plugin's package, so you should now be able to install the Slack plugin in production! Let me know if you run into any issues, or have any other questions.

**Reply by Saauan** [reply]
Thanks ! I installed and tested it and it works as expected :+1:
